pkgs/top-level/all-packages.nix

```diff
@@ -20687,7 +20687,7 @@ with pkgs;
   # These are used when buiding compiler-rt / libgcc, prior to building libc.
   preLibcCrossHeaders = let
     inherit (stdenv.targetPlatform) libc;
-  in     if libc == "msvcrt" then targetPackages.windows.mingw_w64_headers or windows.mingw_w64_headers
+  in     if isMinGW then targetPackages.windows.mingw_w64_headers or windows.mingw_w64_headers
```

Comment:
```
error: undefined variable 'isMinGW'
```

```suggestion
  in     if stdenv.targetPlatform.isMinGW then targetPackages.windows.mingw_w64_headers or windows.mingw_w64_headers
```

Reply:
@OPNA2608 thanks, missed that in the rewrite.